source/conf.py

```diff
@@ -381,8 +381,22 @@
     'user': (f'{github_sponsors_url}/%s', '@'),  # noqa: WPS323
 }
 
+linkcheck_anchors = False
+
 linkcheck_ignore = [
     "http://localhost:\d+",
+    # Ignoring these links as they will redirect to login page if folks haven't logged in.
+    "https://github.com/pypa/packaging-problems/issues/new?/*",
+    "https://console.cloud.google.com/bigquery/*",
+    "https://console.developers.google.com/apis/library/bigquery-json.googleapis.com",
+    "https://test.pypi.org/manage/account/#api-tokens",
```

Comment:
I think most of the places behind auth could use wildcards like
```suggestion
    "https://test.pypi.org/manage/*",
```